### PR TITLE
added active-storage for hospital images and home page for hospitals …

### PR DESCRIPTION
**Pull request by bacancy-priyank**
…w with search using ajax

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -12,123 +10,146 @@
...
+    t.index ["user_id", "role_id"], name: "index_users_roles_on_user_id_and_role_id"
+    t.index ["user_id"], name: "index_users_roles_on_user_id"
   end
 
-  add_foreign_key 'appointments', 'doctors'
-  add_foreign_key 'availabilities', 'doctors'
-  add_foreign_key 'doctors', 'hospitals'
-  add_foreign_key 'orders', 'users'
+  add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
+  add_foreign_key "appointments", "doctors"
```
Comment (pinkal-bacancy) [inline db/schema.rb]: Don't push schema file.

File: app/views/hospitals/index.html.erb [hunk 2]
```diff
@@ -12,6 +13,7 @@
   </tr>
   <% @hospitals.each do |hospital| %>
   <tr>
+     <td ><%= image_tag(hospital.image,style:"width: 50%")%></td>
```
Comment (pinkal-bacancy) [inline app/views/hospitals/index.html.erb]: Why you have added inline css for image?

Comment (bacancy-priyank) [reply app/views/hospitals/index.html.erb]: ema class work nota krta

Comment (bacancy-priyank) [reply app/views/hospitals/index.html.erb]: done

**Pull request merged**